apps/ui/lib/ui_web/schema/threads_queries.ex

```diff
@@ -0,0 +1,25 @@
+defmodule UiWeb.Schema.ThreadsQueries do
+  use Absinthe.Schema.Notation
+
+  object :threads_queries do
+    @desc "Get thread by url"
+    field :thread, :thread do
+      arg(:url, non_null(:string))
+
+      resolve(fn _, %{url: url}, %{context: _} ->
+        Ui.FindThread.find_thread(url)
+      end)
+    end
+  end
+
+  object :threads_mutations do
+    @desc "Create a thread with the args"
+    field :create_thread, type: :thread do
```

Comment:
I don't think this is needed at all, this mutation to create thread, nor query to find thread by URL.

@baalberith can't we just have it created for us behind the scenes? Like, I am interested in one query: "Give me all comments for current URL", and one mutation: "add comment for the current URL", both taking URL as argument and querying / creating thread behind the scenes.

I don't think we ever have to expose the Thread or thread_id to the client, we'd just deal with comments.

Not sure, do I make sense?

Reply:
@hubertlepicki done